.github/workflows/main.yaml

```diff
@@ -29,7 +29,15 @@ jobs:
 
       - name: Build
         run: |
-          go build -v ./...
+          make agent
```

Comment:
There is no particular reason to make service by service in CI - please use just `make` to make all services at once.

Reply:
It is fine now